src/Graphics/Vty/Input/Events.hs

```diff
@@ -43,6 +43,7 @@ data Event
     -- row. Some terminals report only that a button was released
     -- without specifying which one; in that case, Nothing is provided.
     -- Otherwise Just the button released is included in the event.
+    -- Mouse wheel release events are not reported.
```

Comment:
You're right, this line sounds like it's about the middle button.
How about "Scrolling the mouse wheel does not trigger release events."?